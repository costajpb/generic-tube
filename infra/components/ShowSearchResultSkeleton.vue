<template>
    <ol :class="classes.grid">
        <li :class="classes.grid__item" v-for="index in 7" :key="index"></li>
    </ol>
</template>

<style module="classes" lang="postcss">
/* FIXME: reuse grid styling */
    .grid {
        margin-block: var(--size-4);
        list-style: none;
        padding: 0;
        display: grid;
        grid-template-columns: repeat(auto-fill, minmax(100px, 1fr));
        gap: var(--size-2);

        @media (min-width: 700px) {
            margin: 0;
        }
    }

    @keyframes gradient {
        0% {
            background-position: -100px
        }

        40%, 100% {
            background-position: 140px
        }
    }
  
  
  
    .grid__item {
        height: 150px;
        border-radius: var(--radius-2);
        animation: gradient 1500ms infinite linear;
        background: linear-gradient(90deg, var(--gray-3) 0px, var(--gray-2) 40px, var(--gray-3) 80px);
        background-size: 600px;
    }
</style>