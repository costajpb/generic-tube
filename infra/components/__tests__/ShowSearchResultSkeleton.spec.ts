import snapshowWrapper from '@/infra/__tests__/snapshotWrapper'
import ShowSearchResultSkeleton from '../ShowSearchResultSkeleton.vue'

describe('infra/components/ShowSearchResultSkeleton', () => {
  it('should render', () => {
    expect(snapshowWrapper(ShowSearchResultSkeleton)).toMatchSnapshot()
  })
})
