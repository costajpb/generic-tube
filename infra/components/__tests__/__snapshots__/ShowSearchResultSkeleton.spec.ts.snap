// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/ShowSearchResultSkeleton > should render 1`] = `
<ol
  class="_grid_f639ee"
>
  
  
  <li
    class="_wrapper_587f33"
  >
    
    
  </li>
  <li
    class="_wrapper_587f33"
  >
    
    
  </li>
  <li
    class="_wrapper_587f33"
  >
    
    
  </li>
  <li
    class="_wrapper_587f33"
  >
    
    
  </li>
  <li
    class="_wrapper_587f33"
  >
    
    
  </li>
  <li
    class="_wrapper_587f33"
  >
    
    
  </li>
  <li
    class="_wrapper_587f33"
  >
    
    
  </li>
  
  
</ol>
`;
